packages/injectable/src/dependency-injection-container/lifecycleEnum.js

```diff
@@ -1,15 +1,16 @@
-export const nonStoredInstanceKey = Symbol('non-stored-instance-key');
+export const transientLifecycle = Symbol('transient-instance-key');
+export const singletonLifecycle = Symbol('singleton-instance-key');
 
 export default {
   singleton: {
     name: 'Singleton',
-    getInstanceKey: () => 'singleton',
+    getInstanceKey: () => singletonLifecycle,
   },
 
   keyedSingleton: ({ getInstanceKey }) => ({ name: 'Keyed', getInstanceKey }),
 
   transient: {
     name: 'Transient',
-    getInstanceKey: () => nonStoredInstanceKey,
+    getInstanceKey: () => transientLifecycle,
```

Comment:
This should remain as `nonStoredInstanceKey`, because the added information is very relevant at this level of abstraction.